src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+
+import { Todos } from '../../types/todos';
+import { UserInfo } from '../UserInfo';
+import { Users } from '../../types/users';
+
+type Props = {
+  todo: Todos;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title" data-cy="titleInput">
+        {todo.title}
+      </h2>
+
+      <UserInfo user={todo.user as Users} />
```

Comment:
user is possibly null, add check if user exists instead if `as`
```suggestion
      <UserInfo user={todo.user} />
```